ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/perception/RDXContinuousHikingPanel.java

```diff
@@ -279,16 +285,35 @@ else if (publishAndSubscribe) // Case 2
 
    public void renderImGuiWidgets()
    {
-      ImGui.text("The ContinuousHikingProcess must be running");
-      ImGui.text("And the enabled checkbox must be checked");
-      ImGui.text("By holding CTRL the robot will walk forward");
-      ImGui.separator();
-      continuousHikingParametersPanel.renderImGuiWidgets();
+      ImGui.checkbox("Enable Continuous Hiking", continuousHiking);
+      continuousHikingParameters.setStepPublisherEnabled(continuousHiking.get());
+
+      ImGui.checkbox("Square Up To Goal", squareUpToGoal);
+
+      if (ImGui.button("Turn Left 90°"))
+      {
+         turnRobot(Math.PI / 2);
```

Comment:
This is a bit pedantic but...
```suggestion
         turnRobot(Math.PI / 2.0);
```

Reply:
Na thats good, I haven't actually fact checked this but I believe its faster with 2.0